examples/tutorials/nb_python/rigid_object_tutorial.py

```diff
@@ -1,44 +1,73 @@
-# [setup]
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: -all
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.4.2
+#   kernelspec:
+#     display_name: Python 3
+#     name: python3
+# ---
+
+# %%
+# !curl -L https://raw.githubusercontent.com/facebookresearch/habitat-sim/master/examples/colab_utils/colab_install.sh | bash -s
```

Comment:
I notice we hard-code `facebookresearch` `master` here and we also get `master` by default when we `git clone` in colab_install.sh. Is there a way to test a new tutorial as a colab, before merging to master? Let's assume the tutorial is exercising new changes to the habitat-sim python API. Maybe I would modify these places to point at my github fork of habitat-sim?

Reply:
@eundersander You replace it with whatever branch you are working on of course.